src/i18n/DE/grid-resources.ts

```diff
@@ -0,0 +1,132 @@
+  igx_grid_excel_select_all_search_results: 'Wähle alle Suchergebnisse aus',
+  igx_grid_excel_add_to_filter: 'Füge aktuelle Auswahl zum Filter hinzu',
+  igx_grid_excel_blanks: '(Leer)',
+  igx_grid_excel_hide: 'Verstecke Spalte',
+  igx_grid_excel_show: 'Zeige Spalte',
+  igx_grid_excel_pin: 'Hefte Spalte an',
+  igx_grid_excel_unpin: 'Löse Spalte',
+  igx_grid_excel_select: 'Selektiere Spalte',
+  igx_grid_excel_text_filter: 'Text-Filter',
+  igx_grid_excel_number_filter: 'Zahlen-Filter',
+  igx_grid_excel_date_filter: 'Datums-Filter',
+  igx_grid_excel_boolean_filter: 'Boolescher-Filter',
+  igx_grid_excel_custom_filter: 'Benutzerdef. Filter...',
+  igx_grid_advanced_filter_title: 'Erweiterte Filterung',
+  igx_grid_advanced_filter_and_group: '"Und" Gruppe',
+  igx_grid_advanced_filter_or_group: '"Oder" Gruppe',
+  igx_grid_advanced_filter_end_group: 'Beende Gruppe',
+  igx_grid_advanced_filter_create_and_group: 'Erstelle "Und" Gruppe',
+  igx_grid_advanced_filter_create_or_group: 'Erstelle "Oder" Gruppe',
+  igx_grid_advanced_filter_and_label: 'und',
+  igx_grid_advanced_filter_or_label: 'oder',
+  igx_grid_advanced_filter_add_condition: 'Bedingung',
+  igx_grid_advanced_filter_ungroup: 'Lösche Gruppierung',
+  igx_grid_advanced_filter_delete: 'Lösche',
+  igx_grid_advanced_filter_delete_filters: 'Lösche Filter',
+  igx_grid_advanced_filter_initial_text: 'Fange an mit einer Gruppe von Bedingungen, die mit "Und" oder "Oder" verknüpft sind',
+  igx_grid_advanced_filter_column_placeholder: 'Wähle Spalte',
+  igx_grid_advanced_filter_value_placeholder: 'Wert',
+  igx_grid_pinned_row_indicator: 'Angeheftet',
+  igx_grid_hiding_check_all_label: 'Verstecke alle',
+  igx_grid_hiding_uncheck_all_label: 'Zeige alle',
+  igx_grid_pinning_check_all_label: 'Hefte alle an',
+  igx_grid_pinning_uncheck_all_label: 'Löse',
+  igx_grid_toolbar_actions_filter_prompt: 'Filter Spalten-Liste...',
+  igx_grid_toolbar_pinning_button_tooltip: 'Öffne Menü, um Spalten anzuheften',
+  igx_grid_toolbar_hiding_button_tooltip: 'Öffne Menü, um Spalten zu verstecken',
+  igx_grid_toolbar_pinning_title: 'Angeheftete Spalten',
+  igx_grid_toolbar_hiding_title: 'Sichtbare Spalten',
+  igx_grid_toolbar_advanced_filtering_button_tooltip: 'Öffne Dialog zur erweiterten Filterung',
+  igx_grid_toolbar_advanced_filtering_button_label: 'Erweiterte Filterung',
+  igx_grid_toolbar_exporter_button_tooltip: 'Öffne Menü, um zu exportieren',
+  igx_grid_toolbar_exporter_button_label: 'Exportiere',
+  igx_grid_toolbar_exporter_excel_entry_text: 'Exportiere als Excel',
+  igx_grid_toolbar_exporter_csv_entry_text: 'Exportiere als CSV',
+  igx_grid_groupByArea_select_message: 'Wähle alle Reihen der Gruppe aus mit Feldnamen {0} und Wert {1}.',
+  igx_grid_groupByArea_deselect_message: 'Wähle alle Reihen der Gruppe ab mit Feldnamen {0} und Wert {1}.',
+  igx_grid_snackbar_addrow_label: 'Reihe hinzugefügt',
+  igx_grid_snackbar_addrow_actiontext: 'Hinzufügen',
```

Comment:
```suggestion
  igx_grid_snackbar_addrow_actiontext: 'HINZUFÜGEN',
```

Reply:
Why all caps?

Reply:
Because it is an action in the snack bar and making it all caps makes it more similar to the buttons' typographies